pkg/apis/api.acorn.io/v1/imageallowrules.go

```diff
@@ -0,0 +1,34 @@
+package v1
+
+import (
+	v1 "github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type ProjectImageAllowRules v1.ProjectImageAllowRulesInstance
```

Comment:
Two thoughts here.

1. Should this be an `admin` API? Creating `ImageAllowRules` seems to be an administrative adjacent feature similar to `ComputeClasses` and `VolumeClasses`.
2. I know that we're getting rid of the concept of `Project*Instance` resources. As such, should we consider removing this type or is there some reason this type wouldn't be affected?

Reply:
1. I thought so too, but @ibuildthecloud mentioned in the beginning of our discussions, that it should be in the "normal" API but with restricted (get/list) access for non-admin users.

2. Yeah, I saw that, but wasn't sure about the reasoning. Do we have some docs on the decision somewhere? No hard feelings from my side.